crates/biome_formatter/src/token/number.rs

```diff
@@ -6,18 +6,40 @@ use std::num::NonZeroUsize;
 use crate::prelude::*;
 use crate::{CstFormatContext, Format};
 
-pub fn format_number_token<L>(token: &SyntaxToken<L>) -> CleanedNumberLiteralText<L>
+#[derive(Debug, Default, Clone, Copy)]
+pub struct NumberFormatOptions {
+    /// Controls how numbers with trailing decimal zeroes are formatted.
+    ///
+    /// Prettier behaves differently when printing numbers like `x.00000` in different languages:
+    /// - In JavaScript: `x.00000` is printed as `x.0`
+    /// - In CSS: `x.00000` is printed as `x`
```

Comment:
I'm unsure how to interpret this information, because it doesn't explain how Biome behaves. Are we diverging? I don't mind doing so, but we should update this page if we decide to do that https://biomejs.dev/formatter/differences-with-prettier/

Reply:
I believe that this is just elaborating on prettier's behavior. We aren't diverging from prettier here.

Reply:
Yeah, we aren't diverging from Prettier. This comment is to explain that we need this option to match Prettier's behavior.